libs/staging-provider/src/Staging/Workspace/BigQueryWorkspaceStaging.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Keboola\StagingProvider\Staging\Workspace;
+
+class BigQueryWorkspaceStaging extends WorkspaceStaging
+{
+    public static function getType()
+    {
+        return 'bigquery';
+    }
+
+    public function getCredentials()
```

Comment:
Vypada to rozumne, jen vubec nechapu, jak si k ni dosel? Poklus&omyl nebo je to nekde zdokumentovany? Protoze testy tady prochazely i bez toho.

Reply:
Je to specifikum pro bigquery